docs/user/dashboard/tsvb.asciidoc

```diff
@@ -1,126 +1,165 @@
 [[TSVB]]
 === TSVB
 
-*TSVB* requires a date field and supports <<aggregation-reference, most {es} metric aggregations>>, multiple visualization types, custom functions,
-and some math. 
-
-TIP: With *TSVB*, you can visualize the data from multiple series, but only *Timelion* can perform math across layers.
+*TSVB* enables you to visualize the data from multiple data series, supports <<aggregation-reference, 
+most {es} metric aggregations>>, multiple visualization types, custom functions, and some math. To use *TSVB*, you must have a date field. 
 
 [role="screenshot"]
 image::visualize/images/tsvb-screenshot.png[TSVB overview]
 
 [float]
 [[tsvb-required-choices]]
-==== Requirements
+==== Open and set up TSVB
+
+Open *TSVB*, then make sure the required settings are configured.
+
+. On the dashboard, click *Create panel*.
+
+. On the *New visualization* window, click *TSVB*.
+
+. In *TSVB*, click *Panel options*, then make sure the following settings are configured:
+
+* *Index pattern*
+* *Time field*
+* *Interval*
+
+. Select a *Drop last bucket* option. Typically, the last bucket contains partial data.
```

Comment:
Small clarification here.

```suggestion
. Select a *Drop last bucket* option. It is dropped by default because the time filter intersects the time range of the last bucket, but can be enabled to see the partial data.
```